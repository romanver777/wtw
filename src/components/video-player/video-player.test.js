/* eslint-disable @typescript-eslint/no-unsafe-call */
/* eslint-disable @typescript-eslint/no-unsafe-member-access */
/* eslint-disable @typescript-eslint/no-unsafe-assignment */
import React from "react";
import TestRenderer from "react-test-renderer";
import VideoPlayer from "./video-player";

describe("VideoPlayer component", () => {
  const data = {
    trailers: [
      {
        url: "https://www.youtube.com/watch?v=xFxgu0ScjJ0",
        name: "Французский blu-ray/dvd-ролик",
        site: "YOUTUBE",
        size: null,
        type: "TRAILER",
      },
      {
        url: "https://widgets.kinopoisk.ru/discovery/trailer/66627?onlyPlayer=1&autoplay=1&cover=1",
        name: "Трейлер (русский язык)",
        site: "KINOPOISK_WIDGET",
        size: null,
        type: "TRAILER",
      },
    ],
    teasers: [],
  };
  const isFullScreen = false;
  const props = {
    data,
    isFullScreen,
  };
  const TRenderer = TestRenderer.create(<VideoPlayer {...props} />);

  it("should renders correct", () => {
    expect(TRenderer.toJSON()).toMatchSnapshot();
  });
});
