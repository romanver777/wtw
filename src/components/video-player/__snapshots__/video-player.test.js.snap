// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer component should renders correct 1`] = `
<iframe
  allowFullScreen={false}
  frameBorder="10"
  src="https://youtube.com/embed/xFxgu0ScjJ0?autoplay=1&mute=1&controls=0"
  style={
    Object {
      "height": "100%",
      "position": "absolute",
      "top": 0,
      "width": "100%",
      "zIndex": 4,
    }
  }
/>
`;
